bosh-shell/Dockerfile

```diff
@@ -4,7 +4,8 @@ RUN gem install bosh_cli -v 1.3262.26.0 --no-rdoc --no-ri
 
 RUN apk add --update bash openssl openssh-client file git && rm -rf /var/cache/apk/*
 
+COPY bosh_wrapper.sh /usr/local/bin
+COPY bosh_wrapper.rc /
 COPY startup.sh /startup.sh
-COPY bosh-ssh /usr/local/bin
```

Comment:
Is it worth leaving a command called bosh-ssh that outputs a helpful message telling people that it's no longer necessary (could even be an alias)

Reply:
Good idea, done. I used a command because it was easier to set the correct return code.